pkg/proxy/winkernel/hnsV1.go

```diff
@@ -223,3 +224,7 @@ func (hns hnsV1) deleteLoadBalancer(hnsID string) error {
 	_, err = hnsloadBalancer.Delete()
 	return err
 }
+
+func (hns hnsV1) updateEndpointPolicy(hnsID string, policy json.RawMessage) error {
+	return fmt.Errorf("DSR is not supported in V1. Using non DSR instead")
```

Comment:
Change this to updateEndpointPolicy is not implemented for V1. This function should be generic and not specific to DSR